### PR TITLE
Add mentor info to Scala track.

### PR DESCRIPTION
**Pull request by PeterPerhac**

### CODE REVIEW

**Comment by PeterPerhac** [comment]
@rpottsoh Hi Ryan, I wrapped the object in square brackets as requested.
I just followed instructions here https://github.com/exercism/website-copy/blob/master/mentors/README.md#add-your-bio

Perhaps what you said could be added to the instructions?

> The file is meant to be an array of mentors. Square brackets should not be indented.

**Comment by rpottsoh** [comment]
Thanks for making the change.  I'll take a look when I get to a computer.  I  am working on a PR right now to add clarity to the README for the very point you have pointed out; it occurred to me yesterday while I was reviewing this PR.

File: mentors/scala/p.json [hunk 1]
```diff
@@ -1 +1,8 @@
-[]
+[{
```
Comment (rpottsoh) [inline mentors/scala/p.json]:
```suggestion
[
  {
```

File: mentors/scala/p.json [hunk 2]
```diff
@@ -1 +1,8 @@
...
+  "github_username": "PeterPerhac",
+  "name": "Peter Perhac",
+  "link_text": "pete's blog",
+  "link_url": "http://perhac.com/blog/",
+  "avatar_url": "http://perhac.com/blog/assets/photo.png",
+  "bio": "Sucked into programming at the age of 11. Worked with several different tech stacks. Currently I work on Scala projects and tinker with Haskell in my spare moments. I want to help _you_ to fall in love with Scala / FP as much as I did."
+}]
```
Comment (rpottsoh) [inline mentors/scala/p.json]:
```suggestion
  }
]
```

**Comment by sshine** [comment]
I believe `[]`s warrant another level of indentation, but other than that, are there any things that block this from merging?

**Comment by rpottsoh** [comment]
> I believe {}s warrant another level of indentation, but other than that, are there any things that block this from merging?

No, just some simple formatting changes.

**Pull request merged**